src/main/scala/firrtl_interpreter/real/DspRealBlackBox.scala

```diff
@@ -95,7 +97,7 @@ abstract class DspRealTwoArgumentToBoolean extends BlackBoxImplementation {
     }
   }
   def cycle(): Unit = {}
-  def execute(inputValues: Seq[Concrete], tpe: Type, outputName: String): Concrete = {
+  def execute(inputValues: immutable.Seq[Concrete], tpe: Type, outputName: String): Concrete = {
```

Comment:
Same general comment as elsewhere, is `immutable.` necessary?

Reply:
Done